code/modules/mob/living/simple_animal/borer/borer.dm

```diff
@@ -267,63 +272,58 @@
 	update_borer_level()
 
 /mob/living/simple_animal/borer/proc/update_borer_level()
-	if((borer_exp >= 20) && (borer_level < 1))
-		borer_level = 1
-		produced_reagents |= list("inaprovaline", "tricordrazine", "synaptizine", "imidazoline", "hyronalin")
-		abilities_in_host |= list(/mob/living/simple_animal/borer/proc/say_host, /mob/living/simple_animal/borer/proc/whisper_host, /mob/living/simple_animal/borer/proc/commune)
-		abilities_standalone |= list(/mob/living/simple_animal/borer/proc/commune)
-		if(host && !controlling)
-			verbs += /mob/living/simple_animal/borer/proc/say_host
-			verbs += /mob/living/simple_animal/borer/proc/whisper_host
-			verbs += /mob/living/simple_animal/borer/proc/commune
-		if(!host)
-			verbs += /mob/living/simple_animal/borer/proc/commune
-		to_chat(src, SPAN_NOTICE("Congratulations! You've reached Evolution Level 1, new syntesis reagents and new abilities are now available."))
-		max_chemicals += (borer_level * 10)
-		max_chemicals_inhost = max_chemicals * 5
-
-	if((borer_exp >= 40) && (borer_level < 2))
-		borer_level = 2
-		produced_reagents |= list("spaceacillin", "quickclot", "detox", "purger", "arithrazine")
-		abilities_standalone |= list(/mob/living/simple_animal/borer/proc/biograde)
-		abilities_in_control |= list(/mob/living/carbon/human/proc/commune)
-		if(!host)
-			verbs += /mob/living/simple_animal/borer/proc/biograde
-		if(host && controlling && ishuman(host))
-			verbs += /mob/living/carbon/human/proc/commune
-		to_chat(src, SPAN_NOTICE("Congratulations! You've reached Evolution Level 2, new syntesis reagents and new abilities are now available."))
-		max_chemicals += (borer_level * 10)
-		max_chemicals_inhost = max_chemicals * 5
-
-	if((borer_exp >= 80) && (borer_level < 3))
-		borer_level = 3
-		produced_reagents |= list("meralyne", "dermaline", "dexalinp", "oxycodone", "ryetalyn")
-		abilities_standalone |= list(/mob/living/simple_animal/borer/proc/invisible)
-		if(!host)
-			verbs += /mob/living/simple_animal/borer/proc/invisible
-		to_chat(src, SPAN_NOTICE("Congratulations! You've reached Evolution Level 3, new syntesis reagents and new abilities are now available."))
-		max_chemicals += (borer_level * 10)
-		max_chemicals_inhost = max_chemicals * 5
-
-	if((borer_exp >= 160) && (borer_level < 4))
-		borer_level = 4
-		produced_reagents |= list("peridaxon", "rezadone", "ossisine", "kyphotorin", "aminazine")
+	if((borer_exp >= BORER_EXP_LEVEL_1) && (borer_level < 1))
+		var/level = 1
```

Comment:
Why not just pass these as arguments directly to the function?

Reply:
I think this is more readable than having a lot of big lists in the proc.